docs/using/installing/index.rst

```diff
@@ -481,8 +481,10 @@ Enabling the Flocker control service on CentOS 7
 .. task:: enable_flocker_control centos-7
    :prompt: [root@control-node]#
 
-The control service needs to accessible remotely.
-To configure FirewallD to allow access to the control service HTTP API, and for agent connections:
+The control service needs to be accessible remotely.
+You will therefore need to configure FirewallD to allow access to the control service HTTP API, and for agent connections.
```

Comment:
Extraneous comma here (it was extraneous before too, but as long as you're changing the line...)